src/main/java/seedu/address/model/profile/ProfileItem.java

```diff
@@ -0,0 +1,102 @@
+package seedu.address.model.profile;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a Profile Item in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class ProfileItem {
+
+    // Identity fields
+    private final String title;
+    private final String type;
+
+    // Data fields
+    private final Set<Tag> tags = new HashSet<>();
+
```

Comment:
I assume this is a planned extension for the profile items to have tags?

Reply:
Yes, there will be multiple descriptors which can be added similarly to tags.